src/components/DropZone/tests/DropZone.test.tsx

```diff
@@ -469,6 +469,9 @@ function setBoundingClientRect(size: keyof typeof widths) {
         left: 0,
         bottom: 0,
         right: 0,
+        x: 0,
```

Comment:
Updates to the type that `Element.getBoundingClientRect()` returns added these new values so we need to add them to the mock (this crops up a few times) https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/#dom-changes